### PR TITLE
add mem opt settings of deeplabv3+

### PR DESCRIPTION
**Pull request by cjld**
This PR is depends on pr: https://github.com/PaddlePaddle/Paddle/pull/15313

### CODE REVIEW

**Comment by cjld** [comment]
The inplace pr(https://github.com/PaddlePaddle/Paddle/pull/14975) is not merged yet, so I put the norm inplace code in python temporarily.

File: fluid/PaddleCV/deeplabv3+/train.py [hunk 1]
```diff
@@ -91,6 +91,41 @@ def loss(logit, label):
     return loss, label_nignore
 
 
+def _inplace_normalize(program, skip_opt_set):
```
Comment (qingqing01) [inline fluid/PaddleCV/deeplabv3+/train.py]: Put this fucntion into another file ?

Comment (cjld) [reply fluid/PaddleCV/deeplabv3+/train.py]: This is just a monkey patch, I will remove it as soon as the inplace PR merged

File: fluid/PaddleCV/deeplabv3+/README.md [hunk 2]
```diff
@@ -40,13 +40,14 @@ data/cityscape/
...
-tar -xf deeplabv3plus_xception65_initialize.tar.gz && rm deeplabv3plus_xception65_initialize.tar.gz
+# TODO change url to paddle cdn
+wget http://vps.randonl.me:8080/deeplabv3plus_xception65_initialize.tgz
+tar -xf deeplabv3plus_xception65_initialize.tgz && rm deeplabv3plus_xception65_initialize.tgz
 ```
 如果需要最终训练模型进行fine tune或者直接用于预测，请下载我们的最终模型
 ```
-wget http://paddlemodels.cdn.bcebos.com/deeplab/deeplabv3plus.tar.gz
-tar -xf deeplabv3plus.tar.gz && rm deeplabv3plus.tar.gz
+wget http://vps.randonl.me:8080/deeplabv3plus.tgz
```
Comment (qingqing01) [inline fluid/PaddleCV/deeplabv3+/README.md]: 需要将模型上传到CDN上，需要更换URL后merge

Comment (cjld) [reply fluid/PaddleCV/deeplabv3+/README.md]: done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**